cmd/root.go

```diff
@@ -265,21 +265,39 @@ func Execute() {
 	if err == nil {
 		forkedFromRemote = "upstream"
 	}
+	_, err = gitconfig.Local("remote.origin.url")
+	if err == nil {
+		forkedFromRemote = "origin"
+	}
+
 	if forkedFromRemote == "" {
-		// use the remote tracked by the branch if set
+		// use the remote tracked by the default branch if set
 		masterRemote, err := gitconfig.Local("branch.master.remote")
```

Comment:
Probably worthwhile with the big shift from `master` to `main`, to include `branch.main.remote` as a case in here as well.

Reply:
Just pushed a new version of the patch.